src/components/input/bl-input.ts

```diff
@@ -184,17 +184,24 @@ export default class BlInput extends FormControlMixin(LitElement) {
 
   /**
    * Overrides error message. This message will override default error messages
+   * @deprecated use setCustomValidity instead
    */
   @property({ type: String, attribute: "invalid-text", reflect: true })
   set customInvalidText(value: string) {
     this._customInvalidText = value;
     this.setValue(this.value);
   }
 
+  /**
+   * @deprecated
+   */
   get customInvalidText(): string {
     return this._customInvalidText;
   }
 
+  @property({ reflect: true, type: String })
+  error: string = "";
```

Comment:
no need to give type, it's already inferred

Reply:
I don't think this decorator is trained in TypeScript. It doesn't hurt to have it as an extra, bro.

Reply:
it's already inferred as part of TypeScript, no additional work required for decorator. since we don't have explicit types for other values it's better to remove it